index.html

```diff
@@ -1,14 +1,259 @@
+        <section class="line-break">
+          <div class="double-line"></div>
+        </section>
+
+        <h4>Drink to your health!</h4>
+        <div id='image-blog'><img src='\images\katiroll.jpg' alt="Healthy" width="75" height="75" /></div>
+        <p>When possible avoid caffeine and alcohol and carbonated soft drinks, and switch to life-giving,
+          vitality-boosting beverages. RASA offers a whole host of SuperDrinks that will feed body, heart and mind.
+          Start with water, that most basic yet most overlooked drink — drinking lots of warm water through the day
+          helps to rehydrate our system and flush toxins out of the body.</p>
+        <a href="/learn-more.html">
+          <button class="learn-more">Learn more</button></a>
+
+      </section>
+  </section>
+  <!-------Footer-------->
+  <section id="footer">
+    <div class="footer">
+      <h2>Questions or Comments? Get in Touch</h2>
+
+      <!----Contact Form-->
+      <section id="footer_content">
+        <form onsubmit="alert('Message Received');return false">
+          <div class="row">
+            <div class="column">
+              <input type="text" class="form-control" placeholder="Name" required />
+              <input type="email" class="form-control" placeholder="Email address">
+              <textarea rows="6" placeholder="Message" cols="40"></textarea>
+
+              <input type="submit" class="send-message" value="Send Message" />
+            </div>
+
+        </form>
+
+        <!------Social Media------>
+        <div id="contact_info">
+          <p class='footer-para'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt commodi impedit
+            quidem saepe similique consequatur?</p>
+
+          <div class='left_contact'>
+
+            <div><img src="\images\address.svg"><a href="Find us here">
+                <p>1234 Somewhere Road<br>Burlington,VT 00000<br>USA</p>
+              </a></div>
+            <div><img src="\images\phone.svg"><a href="Reach@">
+                <p>(000)-000-0000</p>
+              </a></div>
+            <div><img src="\images\email.svg"><a href="mailto:hege@example.com">contact@example.com</a></div>
+
+          </div>
+
+          <div class="social_contact">
+            <div class="column">
+              <div><img src="\images\twitter.svg">Twitter</div>
+              <div><img src="\images\insta.svg">Instagram</div>
+              <div><img src="\images\facebook.svg">Facebook</div>
+              <div><img src="\images\linkedIn.svg">Linkedin</div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </div>
+
+  </section>
+  <div class="attribution">
+    Icons by <a href="https://freeicons.io/line-hero-essentials/essentials-phone-call-icon-20476">wishforge</a>
+    <a href="https://freeicons.io/co-working-and-office/letter-email-icon-1220">icon king1</a>
+    <a href="https://freeicons.io/profile/823">
+      Muhammad Haq</a>
+    </section>

```

Comment:
We could move this inside of the `footer` and make it smaller along with moving it into one of the corners of the page so that the it's more of a foot note